spec/components/previews/login_button_component_preview.rb

```diff
@@ -0,0 +1,23 @@
+class LoginButtonComponentPreview < ButtonComponentPreview
+  # @!group Preview
+  def default
+    render(LoginButtonComponent.new)
+  end
+
+  # @!endgroup
+
+  # @param big toggle
+  # @param color select [~,primary,primary-darker,primary-lighter]
+  def workbench(
+    big: false,
+    color: 'primary'
+  )
+
+    render(
+      LoginButtonComponent.new(
+        big:,
+        color:,
+      ),
+    )
+  end
+  end
```

Comment:
Some code style nitpicks on whitespace / indentation:

```suggestion
    render(
      LoginButtonComponent.new(
        big:,
        color:,
      ),
    )
  end
end
```